sympy/combinatorics/perm_groups.py

```diff
@@ -3231,9 +3229,9 @@ def is_cyclic(self):
         order = self.order()
 
         if order < 6:
-            self._is_abelian == True
+            self._is_abelian = True
             if order != 4:
-                self._is_cyclic == True
+                self._is_cyclic = True
```

Comment:
I doubt these lines are tested... (Unfortunately, I know nothing about the subject.)

Reply:
docstring affirms
```
        - If the order of the group is less than $6$ and is not $4$, the
          group is cyclic.
```
I don't see assurance that it is Abelian, however. It could be None or True at this point in the code,

Reply:
@sylee957 do you remember how this should work?

Reply:
I think that the new behavior should work if it passes the tests